tools/evolve.js

```diff
@@ -24,4 +24,6 @@ const proposalResult = ao.message({
   data: BUNDLED_AOS_LUA,
   signer,
 });
+if (!proposalResult || typeof proposalResult !== 'string')
```

Comment:
is this an incomplete `if` statement (no brackets)?

Reply:
Nope, valid syntax :) its a "one line" if block... despite being on two lines.